### PR TITLE
Enable CodeQL on Jira Integration

### PR DESCRIPTION
**Pull request by tebriel**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      # Override language selection by uncommenting this and choosing your languages
+      # with:
+      #   languages: go, javascript, csharp, python, cpp, java
```
Comment (srt32) [inline .github/workflows/codeql-analysis.yml:26]: We don't need this line at all?

Comment (tebriel) [reply .github/workflows/codeql-analysis.yml:26]: Nope, looks like it autodetects!

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    # ℹ️ Command-line programs to run using the OS shell.
+    # 📚 https://git.io/JvXDl
+
+    # ✏️ If the Autobuild fails above, remove it and uncomment the following three lines
+    #    and modify them (or add more) to build your code if your project
+    #    uses a compiled language
+
+    #- run: |
+    #   make bootstrap
+    #   make release
```
Comment (srt32) [inline .github/workflows/codeql-analysis.yml:42]: Shall we remove the comments?

Comment (tebriel) [reply .github/workflows/codeql-analysis.yml:42]: We can, I installed this through the marketplace + actions without modification. What do you think is best?

Comment (srt32) [reply .github/workflows/codeql-analysis.yml:42]: I'd propose we delete all the comments but that's just an opinion ;)

Comment (tebriel) [reply .github/workflows/codeql-analysis.yml]: done!

**Pull request merged**